Content.Shared/_EE/Gravity/SharedFloatingVisualizerSystem.Flying.cs

```diff
@@ -0,0 +1,25 @@
+using System.Numerics;
+using Robust.Shared.Map;
+using Content.Shared._EE.Flight.Events;
```

Comment:
Unused using directives
```suggestion
using Content.Shared._EE.Flight.Events;
```